.env.example

```
@@ -0,0 +1,24 @@
+COMPOSE_PROJECT_NAME=open-dpp-api-tests
+
+DB_HOST=
+DB_PORT=
+DB_USERNAME=
+DB_PASSWORD=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_PORT key

(UnorderedKey)